__tests__/integration.js

```diff
@@ -59,6 +59,11 @@ const PORT_RANGE = MAX_PORT_NUM - MIN_PORT_NUM;
 const getRandomPort = () => Math.floor(Math.random() * PORT_RANGE) + MIN_PORT_NUM;
 
 async function runYarn(args: Array<string> = [], options: Object = {}): Promise<Array<Buffer>> {
+  if (!options['env']) {
+    options['env'] = Object.assign({}, process.env);
```

Comment:
Why `Object.assign()` here instead of the spread notation like other places?

Reply:
Agreed. This issue has been fixed.